modules/openapi-generator/src/main/resources/kotlin-client/build.gradle.mustache

```diff
@@ -54,10 +54,12 @@ apply plugin: 'kotlinx-serialization'
 apply plugin: 'idea'
 {{/idea}}
 apply plugin: 'maven-publish'
+{{^useSettingsGradle}}
```

Comment:
@dylan-kwon shouldn't we also remove the settings.gradle file?

Reply:
The `settings.gradle` I am talking about in this PR is `settings.gradle` in the root of the Gradle project that uses `open-apiGenerator`, not `settings.gradle` in a directory automatically generated by `open-apiGenerator`.

Reply:
Thanks 👍